src/app/child-dev-project/children/children.service.ts

```diff
@@ -124,64 +123,6 @@ export class ChildrenService {
     return from(promise);
   }
 
-  getAttendances(): Observable<AttendanceMonth[]> {
-    return from(this.entityMapper.loadType<AttendanceMonth>(AttendanceMonth));
-  }
-
-  getAttendancesOfChild(childId: string): Observable<AttendanceMonth[]> {
-    const promise = this.dbIndexing.queryIndexDocs(
-      AttendanceMonth,
-      "attendances_index/by_child",
-      childId
-    );
-
-    return from(promise);
-  }
-
-  getAttendancesOfMonth(month: Date): Observable<AttendanceMonth[]> {
-    const monthString =
-      month.getFullYear().toString() + "-" + (month.getMonth() + 1).toString();
-    const promise = this.dbIndexing.queryIndexDocs(
-      AttendanceMonth,
-      "attendances_index/by_month",
-      monthString
-    );
-
-    return from(promise);
-  }
-
-  /**
-   * @deprecated use AttendanceService instead. This can be removed after all AttendanceMigrationService tasks are completed.
```

Comment:
the whole `AttendanceMonth` implementation is deprecated (i.e. shouldn't be used anymore) but not irrelevant yet. Please leave this until all active user instances are migrated, parts of this may be needed for migration and troubleshooting during migration.

Reply:
Yes `AttendanceMonth` is still used in the `AttendanceMigrationService`. But the function call in the `ChildrenService` are not used anywhere? I dont really see how this could be used by anyone. And currently the indices are built every time even though nobody uses it.